app-modules/interaction/resources/views/components/interaction-timeline-item.blade.php

```diff
@@ -33,20 +33,22 @@
 --}}
 @use('App\Filament\Resources\UserResource')
 <div>
-    <div class="flex flex-row justify-between">
-        <h3 class="mb-1 flex items-center text-lg font-semibold text-gray-500 dark:text-gray-100">
-            <a
-                class="font-medium underline"
-                href="{{ UserResource::getUrl('view', ['record' => $record->user]) }}"
-            >
-                {{ $record->user->name }}
-            </a>
-        </h3>
+    @if ($record->user)
```

Comment:
Hmm, I could see this fixing issues where a Timeline record does not have a `user` for some reason. But would this not also remove the `$viewRecordIcon` making you unable to view the record in detail?

@danharrin what do you think?

Reply:
Yeah is it possible just to put the user link inside the conditional instead?

Reply:
@danharrin  & @Orrison 
You mean wrap condition on anchor tag. right?

Reply:
Yeah, it just seems like this is covering too much.